origin-bridge/logic/attestation_service.py

```diff
@@ -166,27 +170,40 @@ def verify_phone(country_calling_code, phone, code, eth_address):
         # but it a good precaution to handle any inconsistency between the
         # success field and the status code
         if response.json()['success'] is True:
-            # TODO: determine what the text should be
-            data = 'phone verified'
-            # TODO: determine claim type integer code for phone verification
-            signature = attestations.generate_signature(
-                signing_key, eth_address, TOPICS['phone'], data
-            )
+            data = {
+                 'issuer': ISSUER,
+                 'issueDate': current_time(),
+                 'attestation': {
+                     'verificationMethod': {
+                         'phone': True
```

Comment:
Would it be worthwhile to record the verification method as `call` or `sms` rather than `phone`?

Reply:
Good idea. Added.